packages/tldraw/src/lib/tools/SelectTool/childStates/Idle.ts

```diff
@@ -34,6 +34,28 @@ export class Idle extends StateNode {
 
 	override onPointerMove: TLEventHandlers['onPointerMove'] = () => {
 		updateHoveredId(this.editor)
+
+		const hitShape = this.editor.getShapeAtPoint(this.editor.inputs.currentPagePoint, {
+			hitInside: false,
+			hitLabels: false,
+			margin: HIT_TEST_MARGIN / this.editor.getZoomLevel(),
+			renderingOnly: true,
+		})
```

Comment:
I think you could use this?
```suggestion
		const hitShape = editor.getHoveredShape()
```